lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -63,6 +63,12 @@
     required: false
     default: true
     version_added: "2.1"
+  ignore_tables:
+    description:
+      - A list of table names that will be ignored in the dump in the form database_name.table_name
+    required: false
```

Comment:
required is by-default 'false' so no need to specify explicitly.

Reply:
Hello. Practically every single option already present in this module specifies it. I prefer explicit as it makes it absolutely clear, even to someone like me with practically no experience with this codebase. Given the above I don’t see any point in removing it from just this particular option.

Do you think I should remove it?